functions/exercise14/scriptSpec.js

```diff
@@ -0,0 +1,13 @@
+/// <reference path="../../typings/globals/jasmine/index.d.ts" />
+/*globals getAmountOfCoins*/
+
+describe("Getting  amount of coins ", function () {
+    it("in the amount of 5 are 5", function () {
+        var amountOfCoins = getAmountOfCoins (5);
+        expect(amountOfCoins).toEqual([5]);
+    });
+    it("in the amount of 45 are 25, 10, 10, 1", function () {
```

Comment:
14. Write a JavaScript function to convert an amount to coins. Go to the editor
Sample function : amountTocoins(46, [25, 10, 5, 2, 1])
Here 46 is the amount. and 25, 10, 5, 2, 1 are coins. 
Output : 25, 10, 10, 1